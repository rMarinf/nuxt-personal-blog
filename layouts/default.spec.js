import { shallowMount } from '@vue/test-utils'
import DefaultLayout from './default.vue'

test('DefaultLayout', () => {
  // render the component
  const wrapper = shallowMount(DefaultLayout, {
    stubs: {
      'the-sidebar': true,
      nuxt: true
    }
  })

  expect(wrapper.element).toMatchSnapshot()
})
