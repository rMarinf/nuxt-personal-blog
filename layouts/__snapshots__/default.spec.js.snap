// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`DefaultLayout 1`] = `
<div
  id="page"
>
  <the-sidebar-stub />
   
  <nuxt-stub />
</div>
`;
